src/components/timeline/GlobalTrack.js

```diff
@@ -231,6 +280,7 @@ export default explicitConnect<OwnProps, StateProps, DispatchProps>({
     let localTrackOrder = EMPTY_TRACK_ORDER;
     let localTracks = EMPTY_LOCAL_TRACKS;
     let pid = null;
+    let graphData = [];
```

Comment:
I think this should be `let graphData = null;` rather than empty data.

Reply:
This variable is initialized as null in the new update